pkg/util/util.go

```diff
@@ -105,7 +105,7 @@ func StrToInt32(val string) *int32 {
 	return n32
 }
 
-//get the hash of the string
+// Hash retrieve a encoded string using the md5 encrypt
```

Comment:
Thanks @rafa-acioly .
Please replace retrieve -> retrieves and a -> an .

Reply:
Sorry about that! 😄